Themis/Views/Assessment/AssessmentView.swift

```diff
@@ -401,6 +406,7 @@ struct AssessmentView: View {
                         }
                         .onEnded {_ in
                             if dragWidthRight < minRightSnapWidth {
+                                showCorrectionSidebar = false
                                 dragWidthRight = 0
                             }
```

Comment:
This is missing the update logic for the case where the correction sidebar is opened by dragging. In this case, `showCorrectionSidebar` is not set to `true`. While this does not impact the animation at all (after all, the user drags already), I believe it's better style to have the correct value stored in `showCorrectSidebar` at all times. This change fixes it:
```suggestion
                            if dragWidthRight < minRightSnapWidth {
                                showCorrectionSidebar = false
                                dragWidthRight = 0
                            } else {
                                showCorrectionSidebar = true
                            }
```